.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": [ "es2015-node5", "stage-2", "react" ]
+  "presets": [ "es2015-node5", "stage-2", "react" ],
+  "sourceMaps": "inline"
```

Comment:
It looks like this is only applying to when babel-register is used, we can certainly override that for the actual code we run in task-build.js